source/agora/consensus/data/Block.d

```diff
@@ -74,6 +77,9 @@ public struct BlockHeader
         dg(this.prev_block[]);
         hashPart(this.height, dg);
         dg(this.merkle_root[]);
+        hashPart(this.enrollments.length, dg);
```

Comment:
You shouldn't need to hash the length.

Reply:
I was thinking about whether to put it in or take it out. :)